htdocs/reception/list.php

```diff
@@ -384,6 +397,13 @@
 			}
 
 			if ($objecttmp->id > 0) {
+				if (!isset($rcp->origin_object)) {
+					$rcp->fetch_origin();
+				}
+				if ($rcp->origin_object->multicurrency_code != $objecttmp->multicurrency_code) {
```

Comment:
There is no amount and no currency on a reception so i don't understand why testing currency.
What is important is only qty.
Can you provide a screenshot of what's wrong ?